docs/user/rstudio/ide/guide/code/diagnostics.qmd

```diff
@@ -33,51 +30,51 @@ Similarly, missing arguments (alongside extra, or missing, commas) are reported:
 
-**Warn if variable is defined but not used**
+### Warn if variable is defined but not used
 
 This diagnostic helps to identify is a variable is created but never used. This can be helpful when attempting to clean up old code, or in diagnosing other errors (wherein you believe a particular variable should be used, but isn't).
 
 In the following example, the variable `result` is assigned but never used or returned; instead, the sum is re-computed and then returned.
 
 ![](images/error-unused.png){fig-alt="A screenshot of a function where a defined object is not used, resulting a warning."}
 
-**Provide R style diagnostics (e.g. whitespace)**
+### Provide R style diagnostics (e.g. whitespace)
 
 The style diagnostic checks to see if your code conforms to [Hadley Wickham's style guide](http://adv-r.had.co.nz/Style.html), and reports style warnings when encountered. In particular, the diagnostics engine attempts to identify inappropriate use (or lack thereof) of whitespace.
 
 ![](images/warning-whitespace.png){alt="" fig-alt="A screenshot of x<-2+1 without any spacing - RStudio provides a note for the style issue expecting whitespace"}
 
 Currently, the style diagnostics feature is not user configurable; this may be addressed in an upcoming release.
 
-## **Other Diagnostics**
+## Other Diagnostics
 
 Diagnostics for other languages, such as C / C++, JavaScript, and Python, are also available.
 
 The C / C++ diagnostics engine is powered by `libclang`, and is able to report compiler errors and warnings inline in the source document.
 
-## **Viewing Diagnostics**
+## Viewing Diagnostics
 
 When an error or warning is discovered, it is reported in two places:
 
 1.  The left gutter will show an icon related to the diagnostic's severity
 
 2.  The associated diagnostic will be underlined.
 
-![](images/error-diagnostic-view.png){alt="" fig-alt="A screenshot of R code displaying a yellow warning in the gutter, a popup for result is defined but not used, and a yellow underline bellow the defined but not used element."}
+![](images/error-diagnostic-view.png){fig-alt="A screenshot of R code displaying a yellow warning in the gutter, a popup for result is defined but not used, and a yellow underline bellow the defined but not used element."}
 
-## **Show Diagnostics**
+## Show Diagnostics
 
 These features allow you to control when diagnostics are presented and updated. Diagnostics can either be shown on save, or after a certain amount of keyboard idle time.
 
 ## Project-Level Diagnostics
 
-You can run the diagnostics engine over all R files in the project either using the code wizard menu, or by pressing `CTRL + ALT + SHIFT + P` (`CMD` on Mac):
+You can run the diagnostics engine over all R files in the project either using the code wizard menu, or by pressing <kbd>Ctrl + Alt + Shift + P</kbd> (<kbd>Cmd + Alt + Shift + P</kbd> on Mac):
```

Comment:
Fyi with my desktop Rstudio (2022.07.2 Build 576) the shortcut is listed as `+ D` instead of `+ P`. Did this shortcut change in the last couple months?

Reply:
Confirmed this on the shortcuts page - changed to `D`